changelog.md

```diff
@@ -12,10 +57,11 @@
 * Update French diceware wordlist (#6936)
 * Replace pretty-bad-protocol dependency with vendored version (#6836, #6907)
 * Import Markup and escape from markupsafe (#6964)
-* Update wordlist to remove potentially confusing or offensive terms (#7008, #7021)
+* Update wordlist to remove potentially confusing or offensive terms (#7024, #7021)
 * Validate the submission key,disable Journalist and Source Interfaces if a weak key is found (#7059)
 * Dependency changes:
   * Update cryptography from 41.0.1 to 41.0.3 (#6940)
+  * Upgrade sequioa-openpgpg from 1.16.1 to 1.17.0 (#7041)
```

Comment:
```suggestion
  * Upgrade sequoia-openpgp from 1.16.1 to 1.17.0 (#7041)
```

Reply:
Always finding new ways to typo sequoia.